org.eclipse.jdt.core/eval/org/eclipse/jdt/internal/eval/CodeSnippetEnvironment.java

```diff
@@ -62,7 +62,9 @@ public NameEnvironmentAnswer findType(char[][] compoundTypeName) {
 		if (CharOperation.equals(compoundTypeName, classFile.getCompoundName())) {
 			ClassFileReader binary = null;
 			try {
-				binary = new ClassFileReader(classFile.getBytes(), null);
+				@SuppressWarnings("deprecation")
```

Comment:
?

Reply:
ClassFileReader constructer deprecated, in order to make the Suppress as small as possible introduced local variable to supress